ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/shutdown_interfaces_towards_undeployed_peers.schema.yml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  shutdown_interfaces_towards_undeployed_peers:
+    documentation_options:
+      filename: Fabric Variables
+      table: Uplink Settings
+    type: bool
+    default: false
+    description: |
+      - It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`.
+
+      ```yaml
+      # Use at the host level
+      is_deployed: < true or false or default -> true >
+      ```
+
+      - By default, this will have no impact within the `eos_designs` role. Configs will still be generated by the `eos_cli_config_gen` role and will still be pushed by the `eos_config_deploy_eapi` directly to devices if used.
+      - However, if the `eos_config_deploy_cvp` role is used to push configurations, CloudVision will ignore the devices flagged  as `is_deployed: false` and not attempt to configure them.
+      - If the device is not present in the network due to CloudVision not configuring the device, `eos_validate_state` role will fail tests on peers of the undeployed device trying to verify that interfaces are up.
+      - To overcome this and shutdown interfaces towards undeployed peers, the variable `shutdown_interfaces_towards_undeployed_peers` can be used, satisfying the `eos_validate_state` role interface tests.
+      - Again, this is only an issue if `eos_config_deploy_cvp` is used and the devices are not present in the network.
```

Comment:
Not sure if we should keep the list syntax `-` in the description

Reply:
All descriptions will be reviewed in the next rounds of PRs where we focus on building the documentation.